cnv/cnv_users_guide/cnv-ansible-modules.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// *
+
+[id="cnv-ansible-modules_{context}"]
+= Automating {ProductName} management tasks by using Red Hat Ansible Automation
+include::modules/cnv-document-attributes.adoc[]
```

Comment:
I'm not sure if how CNV does things is different, but I think once you add this module to an assembly, I'd think the assembly should include this attribtues file, and you wouldn't include it here.